superset-frontend/src/views/CRUD/data/database/DatabaseModal/index.tsx

```diff
@@ -1088,7 +1088,7 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
     let alertErrors: string[] = [];
     if (isEmpty(dbErrors) === false) {
```

Comment:
Can we also modify this condition to keep the code readable to be consistent? Thanks!